GliaWidgets/SecureConversations/SecureConversations.ChatWithTranscriptModel.swift

```diff
@@ -436,3 +627,14 @@ extension SecureConversations.TranscriptModel {
         }
     }
 }
+
+// MARK: Hashable
+extension SecureConversations.TranscriptModel: Hashable {
+    func hash(into hasher: inout Hasher) {
+        hasher.combine(ObjectIdentifier(self))
+    }
+
+    static func == (lhs: SecureConversations.TranscriptModel, rhs: SecureConversations.TranscriptModel) -> Bool {
+        lhs === rhs
+    }
```

Comment:
I think this is not needed, because after `hash` func implementation, `Equatable` protocol will use hash sum for comparing objects...

Reply:
`Hashable` is dependent on `Equatable` so implementation is required. Also it so not correct to assume that objects are equal only by comparing their hash values.